stories/constants/colors.stories.tsx

```diff
@@ -1,54 +1,26 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import { SPACER } from 'src/constants';
-import { Typography } from 'src/shared-components';
-import * as ALL_COLORS from 'src/constants/colors';
+import { useTheme } from 'emotion-theming';
 
 import Color from './color';
```

Comment:
I think this story is no longer appropriately nested in `/constants/`, since we're removing the export. We'll also need to update `constants/index.stories.tsx` to update the usage example: `import { BOX_SHADOWS, COLORS, SPACER } from 'radiance-ui/lib/constants';` 

I don't want this to be a blocker, especially since we're relying on Ben to double-check values on that route, so let's make a follow-up item (to be done in a future PR) about re-arranging our stories to reflect usage for v14.

Reply:
created a task to fix the stories of Colors, box shadows and fonts once they are forced into theme